backend/index.js

```diff
@@ -52,6 +54,8 @@ app.use("/", homeRoutes); // example stuff for testing
 
 // Routes with JWT check middleware
 app.use("/nested-path-example/", checkJwt, helloRoute); // example stuff for testing
+app.use("/api/", parkRoutes);
+app.use("/api/", seasonRoutes);
```

Comment:
I mentioned leaving out the "/api/" part from the path on Slack, but I actually don't know what we're going to do with the backend server in production, so let's leave it in, just to be safe 😇